programs/server/config.xml

```diff
@@ -714,6 +714,15 @@
     <!-- <path_to_regions_names_files>/opt/geo/</path_to_regions_names_files> -->
 
 
+    <!-- Custom TLD lists.
+         Format: <name>/path/to/file</name>
+    -->
+    <top_level_domains_lists>
+        <!--
+        <public_suffix_list>/path/to/public_suffix_list.dat</public_suffix_list>
```

Comment:
Updating a list file requires server restart, right?

Removing a list from configuration looks problematic -- the reference to it from a simultaneously running function may become dangling.

Reply:
>Updating a list file requires server restart, right?

Correct

>Removing a list from configuration looks problematic -- the reference to it from a simultaneously running function may become dangling.

Actually this can be supported by storing shared_ptr, but this can be done later I guess.

Reply:
Somehow I thought we already can remove the list w/o restart, but apparently not, so no problem with dangling references.